src/redisjson.rs

```diff
@@ -490,6 +509,10 @@ pub mod type_methods {
             }
             3 => {
                 let data = raw::load_string(rdb);
+                if raw::load_unsigned(rdb) > 0 {
+                    raw::load_string(rdb);
+                    raw::load_string(rdb);
+                };
```

Comment:
@rafie @oshadmi these lines are only needed on version 2

Reply:
@gkorland It was added in PR [#333](https://github.com/RedisJSON/RedisJSON/pull/333)
And merged to here from master.
I will remove it.